sdk/communication/communication-phone-numbers/src/generated/src/operations/phoneNumbers.ts

```diff
@@ -156,7 +156,7 @@ export class PhoneNumbers {
     options?: PhoneNumbersPurchasePhoneNumbersOptionalParams
   ): Promise<LROPoller<PhoneNumbersPurchasePhoneNumbersResponse>> {
     const operationArguments: coreHttp.OperationArguments = {
-      options: this.getOperationOptions(options, "location")
+      options: this.getOperationOptions(options, "undefined")
```

Comment:
Let's make sure that this still works with generated LROs to get the final search result